File: frontend/src/app/dto/class-dtl/class-dtl.insert.res.dto.ts
import { ClassDtlInsertDataResDto } from "./class-dtl.insert.data.res.dto";

export class ClassDtlInsertResDto{
    private _data! : ClassDtlInsertDataResDto
	private _message! : string

    public get data(): ClassDtlInsertDataResDto {
        return this._data;
    }

    public set data(data: ClassDtlInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}

File: src/main/java/com/lawencon/elearn/dto/classdtl/ClassDtlInsertResDto.java
package com.lawencon.elearn.dto.classdtl;

public class ClassDtlInsertResDto {
	private ClassDtlDataResDto data;
	private String message;
	public ClassDtlDataResDto getData() {
		return data;
	}
	public void setData(ClassDtlDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
